### ISSUE TITLE
Main and develop branches are diverging

### ISSUE THREAD

**Issue by Penaz91** [Vinetos/Tranquille#18]
From a cursory examination, it seems that the main and develop branches are diverging slightly.

This may be an issue when merging is involved, so I think it may be wise to realign the two branches while they can still be merged automatically.

**Comment by Vinetos** [comment]
Hello !
Could you elaborate a little more ? I think I do not understand your point 

I think you are saying that `develop` is something like 9 commits ahead from `main`. If so, here is an explanation : 

I have chosen to follow the Git Flow organization : ![Git flow graphic](https://miro.medium.com/v2/resize:fit:1400/1*3-0EDzE63S_UZx2KbIz_dg.png)


`main` is the _current release_ branch. This can be considered as the "stable" version of the application, available for all users. This a way to provide hotfix for the version of the app that is deployed on app stores without having to do weird and hard-to-follow git manipulation.

 `develop` is the _next release_ branch. This branch contains all the feature that will be shipped with the next release of the application. This allows beta-testers to report bug before releasing the app to everyone and needing to create a hotfix.
 
When I print the repository graph, I do not see any problem : 
```shell
# git log --graph --pretty --abbrev-commit --oneline
* 11ce8f1 (HEAD -> develop, origin/develop) fix: nb-NO and pt-BR fastlane locales (#17)
*   e20ee5e Fix deprecated build options (#9)
|\  
| * 5b15f41 Update Changelog
| * f10ca69 Upgrade Android gradle plugin to v8.3.1
| * 11f0411 Update build options (Fixes #8)
* | a57102f Fix some FDroid-Bot Suggestions (#7)
|\| 
| * c529977 Updated Changelog
| * 90abbda User the complete gradle zip
| * 224127a Fix gradle wrapper version mismatch
| * fffb189 Fixed wrong fastlane folder names
| * 0406588 Added icon.png to fastlane metadata
|/  
*   dd300c6 (tag: 0.0.1, origin/main, origin/HEAD, main) Release 0.0.1 (#5)
```

User willing to contribute have to create their branch based on `develop` if they create a feature, a bugfix or whatever that must not be immediately released to everyone. 

Was it your point ? I know I have to update `CONTRIBUTING.MD` to include these explanation but #6 is taking a lot of my compute time haha ! 

Credit to https://medium.com/@yanminthwin/understanding-github-flow-and-git-flow-957bc6e12220 for the graph.

**Comment by Penaz91** [comment]
It seems it has been a misreading on my end. From what I saw it looked like !17 was merged into main but not devel, hence my question.

I wonder if it was a caching issue or something, since things look different now. :facepalm: 

Sorry for raising an undue warning.
